### PR TITLE
Starforge Haks assets

### PR DESCRIPTION
**Pull request by Lamarr13**
Starforge Haks assets

### CODE REVIEW

File: swlor2_mdl/c_fcfield.mdl [hunk 1]
```diff
@@ -0,0 +1,401 @@
+# NeverBlender 4.0.0|Blender 4.0.2|2023-12-30
```
Comment (zunath) [inline swlor2_mdl/c_fcfield.mdl:1]: All models must be compiled.

**Review by zunath — CHANGES_REQUESTED** [review]
Requesting changes above

File: swlor2_mdl/c_fcfield.mdl (hunk 1 above)
Comment (Lamarr13) [reply swlor2_mdl/c_fcfield.mdl]: MDLs should be compiled, removed mdl that are in the creature pack

File: swlor2_mdl/dr_lehon01.mdl [hunk 2]
```diff
@@ -0,0 +1,4672 @@
+# NeverBlender 4.0.0|Blender 4.0.2|2023-12-25
```
Comment (zunath) [inline swlor2_mdl/dr_lehon01.mdl:1]: More compilation stuff. Make sure you're looking at all the individual files here so you know what's been compiled and what hasn't: https://github.com/zunath/SWLOR_Haks/pull/241/files

**Comment by zunath** [comment]
Merging in the other creature changes caused a conflict in this PR. Please review & correct when available.

**Pull request closed without merging**